themes/default/css/style.css

```diff
@@ -26,6 +26,12 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+/* HiDPI */
+@media screen and (min-width: 3000px)
+{
+    html { zoom: 2.0; }
+}
```

Comment:
Could you format this using the same style that the rest of the file uses?
eg.
```css
@media screen and (min-width: 3000px) {
    html {
        zoom: 2.0;
    }
}
```

Thanks!

Reply:
Alrighty, fixing it right away. Forgot to add min-height anyway. :)